app/assets/javascripts/header.js

```diff
@@ -94,4 +68,36 @@ function setHeight() {
   $('#header_space').css({"height": the_height});
 }
 
+function expandButton(event) {
+  hideSmallTopNav();
+  console.log(event.data);
+  if (event.data.value % 2 == 0) {
```

Comment:
As it's not straightforward to rename this variable (without pouting global namespace) maybe a comment will do? 

```javascript
function expandButton(event) {
  hideSmallTopNav();
  // even.data.value -> how many times was this function called?
  // if even number of times -> show expand_me
  // otherwise -> hide expand_me
  if (event.data.value % 2 == 0) {
    showExpandMe();
  } else {
    hideExpandMe();
  }
  setHeight();
  event.data.value++;
}
````